DEFAULT_CONFIG.json5

```diff
@@ -330,7 +330,17 @@
       /// If not configured, all the supported protocols are automatically whitelisted.
       /// The supported protocols are: ["tcp" , "udp", "tls", "quic", "ws", "unixsock-stream", "vsock"]
       /// For example, to only enable "tls" and "quic":
-      //   protocols: ["tls", "quic"],
+      ///   protocols: ["tls", "quic"],
+      ///
+      /// ## Endpoint metadata
+      ///
+      /// **priorities**: a Rust-style half-open range (e.g. `2..4` signifies priorities 2 and 3).
```

Comment:
The range should be inclusive. It doesn't make sense to express `priorities=1..9` to set any priority from `1` to `8` given that priority `9` does not exist.

Reply:
Good point!